util_scripts/dev-api-encoded-secret.sh

```diff
@@ -32,29 +36,37 @@ echo "---------------------------------------------------------------"
 # curl -X GET --location  "$COMMUNITY_HOST/secure/staff/staffIdentifier/2500012436" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
-
-
 # curl -X GET --location  "$COMMUNITY_HOST/secure/staff/username/lucybennett" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
 # curl -X GET --location  "$COMMUNITY_HOST/secure/staff/username/timharrison" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
-
 # curl -X GET --location  "$COMMUNITY_HOST/secure/offenders/crn/X356288/allOffenderManagers" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
 # curl -X GET --location  "$COMMUNITY_HOST/secure/offenders/crn/X356288/contact-summary" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
 # curl -X GET --location  "$COMMUNITY_HOST/secure/staff/staffIdentifier/2500002049/managedOffenders?current=true" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
 
 # Prison register
-# curl -v --location  "$PRISON_REGISTER_HOST/prisons/id/PBI" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
-# curl -v --location  "$PRISON_REGISTER_HOST/secure/prisons/id/PBI/offender-management-unit/email-address" -H "Authorization: $AUTH_TOKEN" -H "Accept: text/plain"
 
-# CVL API
-curl -v --location  "$CVL_API/licence/match?prison=MDI&status=APPROVED&nomsId=A7814DY" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
+# curl -v -X GET "$PRISON_REGISTER_HOST/prisons" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
+# curl -v -X GET "$PRISON_REGISTER_HOST/secure/prisons/id/CFI/offender-management-unit/email-address" -H "Authorization: $AUTH_TOKEN" -H "Accept: text/plain"
+# curl -v -X PUT "$PRISON_REGISTER_HOST/secure/prisons/id/SWI/offender-management-unit/email-address" -d "test@test.com" -H "Authorization: $AUTH_TOKEN" -H "Content-Type: text/plain" -H "Accept: text/plain"
+# curl -v -X GET "$PRISON_REGISTER_HOST/secure/prisons/id/SWI/offender-management-unit/email-address" -H "Authorization: $AUTH_TOKEN" -H "Accept: text/plain"
+# curl -v -X GET "$PRISON_REGISTER_HOST/secure/prisons/id/UKI/offender-management-unit/email-address" -H "Authorization: $AUTH_TOKEN" -H "Accept: text/plain"
+# curl -v -X GET "$PRISON_REGISTER_HOST/secure/prisons/id/UPI/offender-management-unit/email-address" -H "Authorization: $AUTH_TOKEN" -H "Accept: text/plain"
+# curl -v -X GET "$PRISON_REGISTER_HOST/secure/prisons/id/PRI/offender-management-unit/email-address" -H "Authorization: $AUTH_TOKEN" -H "Accept: text/plain"
+
+# Create and vary a licence API
+
+# curl -v -X GET "$CVL_API/licence/match?prison=MDI&status=APPROVED&nomsId=A7814DY" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
+# curl -X GET "$CVL_API/config/pdu-contacts" -H "Authorization: $AUTH_TOKEN" -H "Accept: application/json" | jq .
```

Comment:
I'd recommend using a postman collection, but I know you are old school 😆

Reply:
Either / Or :-)